src/extensions/extension-discovery/extension-discovery.test.ts

```diff
@@ -114,13 +117,14 @@ describe("ExtensionDiscovery", () => {
         },
         manifestPath: path.normalize("some-directory-for-user-data/node_modules/my-extension/package.json"),
       });
-      done();
+      letTestFinish.set(true);
```

Comment:
Do we really need this? If the motivation is that you are not allowed to have `async` and `done` in same test, could we just remove `async` since it wasn't in use at all? :)